examples/2-query/src/Monster.re

```diff
@@ -29,81 +30,78 @@ module GetPokemon = [%graphql
-                     switch (
-                       pokemon##image,
-                       pokemon##classification,
-                       pokemon##name,
-                       pokemon##height,
-                       pokemon##weight,
-                     ) {
-                     | (
-                         Some(image),
-                         Some(classification),
-                         Some(name),
-                         Some(height),
-                         Some(weight),
-                       ) =>
-                       <section className=Styles.dexContainer>
-                         <div className=Styles.dex>
-                           <div className=Styles.dexImageContainer>
-                             <img className=Styles.dexImage src=image />
-                           </div>
-                           <div className=Styles.dexText>
-                             <h1 className=Styles.dexTitle>
-                               name->ReasonReact.string
-                             </h1>
-                             <h2 className=Styles.dexSubTitle>
-                               classification->ReasonReact.string
-                             </h2>
-                             {
-                               switch (height##maximum, weight##maximum) {
-                               | (Some(heightMax), Some(weightMax)) =>
-                                 <div className=Styles.dexGrid>
-                                   <p>
-                                     ("Height: " ++ heightMax)
-                                     ->ReasonReact.string
-                                   </p>
-                                   <p>
-                                     ("Weight: " ++ weightMax)
-                                     ->ReasonReact.string
-                                   </p>
-                                 </div>
-                               | (_, _) => ReasonReact.null
-                               }
-                             }
-                           </div>
-                         </div>
-                       </section>
-                     | (_, _, _, _, _) => ReasonReact.null
-                     }
-                   | None => ReasonReact.null
-                   }
-                 | Fetching => <div> "Loading"->ReasonReact.string </div>
-                 | Error(error) => <div> "Error"->ReasonReact.string </div>
-                 | NotFound => <div> "Not Found"->ReasonReact.string </div>
-                 }
-               }
-             </>
-         }
-    </Query>;
-  },
-};
\ No newline at end of file
+[@react.component]
+let make = (~pokemon: string) => {
+  /* We set up the query here as we need access to the pokemon
+      value passed in from GetAll */
+  let request = GetPokemon.make(~name=pokemon, ());
+  let query = request##query;
+  let variables = request##variables;
+  let ({response}, _executeQuery) = useQuery(~query, ~variables, ());
+
+  <>
```

Comment:
Is this `Fragment` necessary? AFAICT pattern matching on `response` should only return a single parent node in the way it's currently written.